### ISSUE TITLE
Borderlands 3 Support

### ISSUE THREAD

**Issue by positiveinflux** [antonpup/Aurora#1860]
I hope i put this request in the right place. I saw most logitech supported games should work when looking through profiles. If this is already working kindly lead me in the right direction. Really Appreciate this project. With all the nice effects with borderlands on my g910 they are sadly very much missed since i now have a corsair k95.  I am very thankful for this program and can't express it enough.

**Comment by Tenshinen** [comment]
Would also appreciate this! Attempting to do it manually by adding the .exe to Aurora doesn't work so...

**Comment by diogotr7** [comment]
I'll try and find out if the game is even trying to load foreign dll's or no.

**Comment by GarettTracer** [comment]
Hello.  I was hoping there may be some kind of update on how the project is going.  Would love to get my Razer devices to work with this in conjunction with my Logitech devices.  Thank you.

**Comment by ZangLucarson** [comment]
Hello, I would also like to know if any progress has been made on attempting to get this game to be compatible with Aurora?
Thank you.

**Comment by adumbidiot** [comment]
Apologies in advance if you already work around this issue, it just seemed similar to something I ran into.

I've was coding my own RGB controller/interceptor for Logitech devices a while back and I ran into problems like this where some games refused to load the patched DLL. I tracked this down to an issue with the default Logitech SDK. Some games load functions from the DLL directly without the Logitech SDK and work fine. However, the Logitech SDK seems to try to verify the authenticity of the DLL before loading with [`WinVerifyTrust`](https://docs.microsoft.com/en-us/windows/win32/api/wintrust/nf-wintrust-winverifytrust). Basically, this means that the DLL must be signed in a way that the computer trusts. Furthermore, I believe Logitech runs some extra checks on the certificate chain as well, ensuring that the CN for both the DLL's certificate and the CA is "Logitech Inc". I was able to work around by making a fake CA with that name and trusting it, then signing my DLL before loading it.

Edit 5/17/2026:
Got deleted pretty fast but there was a user who posted malware that was supposedly a guide on how to do the aforementioned steps. These steps can compromise your computer's security if you don't know what you are doing, so never follow any guides on how to do them.

**Comment by diogotr7** [comment]
Yes, this is most likely what is happening. Same thing with Razer Chroma games.
